.gitignore

```diff
@@ -6,6 +6,7 @@ TestResults/
 _ReSharper.*/
 packages/
 artifacts/
+fakeinstall/
```

Comment:
It is just a temporary place-holder for a proper install path. What path would you prefer on Windows (similar to ~/.omnisharp/local)?